monkey/tests/unit_tests/infection_monkey/telemetry/test_exploit_telem.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from infection_monkey.exploit.wmiexec import WmiExploiter
+from infection_monkey.exploit.sshexec import SSHExploiter
```

Comment:
Why are we changing the exploiter ?

Reply:
See commit messages: https://github.com/guardicore/monkey/pull/1336/commits/2496ed08896e815637ff72884ba90492d429cd88